content/docs/command-reference/diff.md

```diff
@@ -149,7 +149,7 @@ files summary: 1 added, 0 deleted, 0 modified
 
 ## Example: Comparing tags or branches
 
-<details>
+<details id="example-comparing-tags-or-branches-expand">
```

Comment:
```suggestion
<details id="example-tags-branches-set-up-the-example">
```

Reply:
@jorgeorpinel maybe remove `example` from the prefix?
`<details id="example-tags-branches-set-up-the-example">`
to
<details id="tags-branches-set-up-the-example">

Reply:
Similar answer: `example-tags-branches-set-up ` ?